### DISCUSSION TITLE
The plugin doesn't work

### DISCUSSION THREAD

**Discussion by noydov** [subhra74/xdm#944] [General]
Asking for help - the plugin tries to load but fails and the plugin doesn't work. I get an XDM Browser Monitor error
Browser------- monitoring is disabled

Please make sure XDM is running and you have the latest version of XDM installed, This addon is compatible with XDM 7.0 or higher
Everything I try to do to fix it doesn't work
Thanks to the respondents

**Comment by frank392a** [comment]
https://github.com/subhra74/xdm-experimental-binaries/releases/tag/8.0.14-beta
